.travis.yml

```diff
@@ -1,30 +1,37 @@
 language: node_js
-
 node_js:
-  - "0.10"
-  - "0.12"
   - "4.2"
   - "5.0"
 
-# Use container-based Travis infrastructure.
 sudo: false
 
+addons:
+  ssh_known_hosts:
+  - 192.241.218.94
+
 branches:
   only:
-    - master
+    - task-travisDeploy
 
 before_install:
+  - openssl aes-256-cbc -K $encrypted_5b6e06e2bfa9_key -iv $encrypted_5b6e06e2bfa9_iv
+    -in deploy_static.pem.enc -out deploy_static.pem -d
+
   # GUI for real browsers.
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
-
-  # Install npmv3 because of postinstall bugs in npmv2:
-  # See https://github.com/FormidableLabs/victory/issues/98
   - npm install -g npm@3
 
 script:
   - npm test
+  - NODE_ENV=production node_modules/.bin/builder run build-static
 
-  # Prune deps to just production and ensure we can build the site
-  - npm prune --production
-  - npm run build-static
+after_success:
```

Comment:
Note to self: change this to deploy only on `master`, e.g. https://github.com/FormidableLabs/victory-docs/blob/master/.travis.yml#L34